### PR TITLE
Provide feedback for ruby gem updates

### PR DESCRIPTION
**Pull request by uberhacker**

### CODE REVIEW

File: packages/ruby.rb [hunk 1]
```diff
@@ -63,6 +63,8 @@ def self.install
   end
 
   def self.postinstall
-    system 'gem update --silent -N --system'
+    puts 'Updating ruby gems. This may take a while...'
```
[the remark this answers is not in the document]
Comment (uberhacker) [reply packages/ruby.rb:66]: Added.

File: packages/ruby.rb [hunk 2]
```diff
@@ -63,6 +63,11 @@ def self.install
...
+    if `grep -c "gem: --no-document" #{HOME}/.gemrc`.to_i.zero?
```
Comment (supechicken) [inline packages/ruby.rb:67]: Check for exit status instead?
```ruby
if system 'grep', '-q', "gem: --no-document", "#{HOME}/.gemrc"
```

File: packages/ruby.rb [hunk 3]
```diff
@@ -63,6 +63,11 @@ def self.install
...
+      system "echo 'gem: --no-document' >> #{HOME}/.gemrc"
```
Comment (supechicken) [inline packages/ruby.rb:68]: Rubyize with `File.write("#{HOME}/.gemrc", "gem: --no-document\n", mode: 'a')`

**Comment by satmandu** [comment]
Added @supechicken's added changes. @supechicken please test and merge if that works...

**Comment by supechicken** [comment]
Tested on `x86_64`

**Pull request merged**